py_gtfs_rt_ingestion/tests/test_configuration.py

```diff
@@ -0,0 +1,67 @@
+import pytest
+
+from datetime import datetime
+from py_gtfs_rt_ingestion import Configuration
+
+def test_filname_parsing():
```

Comment:
all of these tests should be straight forward. the only semi neat thing in here is using `pytest.raises` to check that bad things to raise exceptions.

these tests should be expanded on later.